Mathlib/RingTheory/DedekindDomain/Ideal.lean

```diff
@@ -1180,7 +1180,7 @@ theorem Ideal.count_normalizedFactors_eq {p x : Ideal R} [hp : p.IsPrime] {n : 
     (normalize_eq _) (Ideal.dvd_iff_le.mpr hle) (mt Ideal.le_of_dvd hlt)
 
 /-- The number of times an ideal `I` occurs as normalized factor of another ideal `J` is stable
-  when regarding at these ideals as associated elements of the monoid of ideals.-/
+  when regarding at these ideals as associated elements of the monoid of ideals. -/
```

Comment:
Is this correct (I'm not fully sure). In any case, the current grammar seems off.
```suggestion
 when regarding these ideals as associated elements of the monoid of ideals. -/